test-cypress/integration/explore/class.hierarchy.spec.js

```diff
@@ -143,6 +143,42 @@ describe('Class hierarchy screen validation', () => {
             });
     });
 
+    it('Test domain range graph', () => {
+        let className = ':Region';
+        searchForClass(className);
+        cy.wait(500);
+        cy.get('.btn-primary').click();
+        cy.wait(500);
+        getDomainRangeGraphHeader().should('contain','Domain-Range graph');
+        getLegendContainer().should('be.visible');
+        getLegendContainer().should('contain', 'main class node');
+        getLegendContainer().should('contain', 'class node');
+        getLegendContainer().should('contain', 'collapsed property');
```

Comment:
I think you can chain the above like:
```
getLegendContainer().should('contain', 'main class node')
.and('contain', 'class node')
.and('contain', 'collapsed property');
```
Same applies for the checks below. The idea is to make the selection once and not to repeat it again and again without need.

Reply:
Chained as shown in commit beb405691242fb594436842557e670b96b7e6a8b